resources-test/build.gradle.kts

```diff
@@ -11,6 +11,12 @@ plugins {
     id("publication-convention")
 }
 
+kotlin {
+    watchosX64()
+    watchosArm64()
+    watchosSimulatorArm64()
```

Comment:
just enable this targets in resources-build-logic/src/main/kotlin/multiplatform-library-convention.gradle.kts

Reply:
convention plugins was reworked